src/Model/Scenario/Properties.hs

```diff
@@ -5,7 +5,7 @@
 {-# LANGUAGE JavaScriptFFI #-}
 {-# LANGUAGE OverloadedStrings #-}
 module Model.Scenario.Properties
-    ( Properties, PropName(..), PropValue
+    ( Properties, PropName(..), PropValue(..)
```

Comment:
Sorry, but I did not expose its constructor intentionally. Use `PToJSVal` or `PFromJSVal` instead.